app/styles/app.css

```diff
@@ -0,0 +1,3 @@
+.ml10 {
+    margin-left: 10px;
+}
```

Comment:
If we want to make this change, we should aim to make it on the style add-on: https://github.com/toddjordan/ember-cli-tutorial-style